pipeline/vars/node_bootstrap.bash

```diff
@@ -31,6 +33,11 @@ python3 -m venv .venv
 source .venv/bin/activate
 python -m pip install --upgrade pip
 python -m pip install -r requirements.txt
+.venv/bin/pip install git+https://gitlab.cee.redhat.com/ccit/reportportal/rp_preproc.git@rpv5
 deactivate
 
+# Install rclone
+curl https://rclone.org/install.sh | sudo bash || echo 0
+wget http://magna002.ceph.redhat.com/cephci-jenkins/.ibm-cos.conf -O ~/.config/rclone/rclone.conf
```

Comment:
You would need to perform `mkdir -p ~/.config/rclone` before copy the file here.

Reply:
IMO, rclone install puts up rclone directory. However I could add initial directory.